blog/urls.py

```diff
@@ -0,0 +1,19 @@
+from django.urls import path
+from blog.views import PostListView, PostDetailView, PostCreateView
+
+
+urlpatterns = [
+    path("", PostListView.as_view(), name="index"),
+    path(""
+         "post/detail/<int:pk>/",
+         PostDetailView.as_view(),
+         name="post-detail"
+         ),
+    path(""
+         "post/create/<int:pk>/",
+         PostCreateView.as_view(),
+         name="post-create"
```

Comment:
just note that we always start endpoints in plural form(like in REST despite it's a default django implementation)